src/test/java/com/infobip/api/MessagesApiTest.java

```diff
@@ -497,7 +572,6 @@ void shouldParseReceiveIncomingMessages() {
                 expectedDestination,
                 expectedText,
                 expectedCleanText,
-                expectedMessageCount,
```

Comment:
Also remove the line where expectedMessageCount variable is declared and initialized as it is no longer used.